raster3d/r3.info/r3.info.md

```diff
@@ -2,6 +2,226 @@
 
 *r3.info* reports metadata and range of 3D voxel maps.
 
+## NOTES
+
+For backwards compatibility reasons, using flags **-g**, **-r** without
```

Comment:
Let's reword this to.

~For backwards compatibility reasons, using flags **-g**, **-r** without
**format**, automatically switches to the **shell** format. In the next
major release, this may be changed to keep the **plain** format output to best
serve interactive use. Specify format **shell** or **json** explicitly for
scripting.~

For backward compatibility, using the **-g ** or **-r** flags without specifying a format currently defaults to the **shell** format. In the next major release, this behavior may change to default to the **plain** format to better support interactive use. To ensure consistent behavior in scripts, explicitly specify the **shell** or **json** format.